variables.tf

```diff
@@ -141,3 +141,7 @@ variable "vpc_all" {
   type        = string
   description = "The full name of the VPC (including environment) used to create resources"
 }
+
+locals {
+  is-production = substr(terraform.workspace, length(var.app_name), length(terraform.workspace)) == "-production"
```

Comment:
this should work, but using `[endswith](https://developer.hashicorp.com/terraform/language/functions/endswith)` function seems cleaner and less prone to error